src/extensions/ExtensionManager.as

```diff
@@ -571,6 +572,13 @@ public class ExtensionManager {
 					if (ext.busy.indexOf(id) == -1) ext.busy.push(id);
 				}
 				break;
+			case '_result':
+				i = value.indexOf(' ');
+				if (i == -1) i = value.length;
+				var resid:int = parseInt(value.slice(0, i));
+				var retval:String = value.slice(i + 1);
```

Comment:
It may be possible to simplify this by using `value.split` as in the `'_busy'` case.